docs/devel/networking.md

```diff
@@ -326,3 +326,41 @@ Chain POSTROUTING (policy ACCEPT)
 target     prot opt source               destination
 MASQUERADE  all  --  10.0.2.2             anywhere
 ```
+
+### Masquerade binding using IPv6 addresses
+The masquerade binding mechanism is currently the only binding mechanism which
+accepts IPv6 addresses.
+
+It operates in exactly the same way as in IPv4, and follows the same goals:
+configure one to one NAT. As in it's IPv4 counter-part, the pods are reached
+via their IPv6 pod addresses.
+
+NAT is configured in the exact same way, but using ip6tables, or the `ipv6-nat`
```

Comment:
Should we mention that there is no DHCP server here?

And do we always forward both IPv4 and IPv6. If not, how do we select the right one?

Reply:
It's actually a pretty important thing, I agree w/ you on both.